package.json

```diff
@@ -112,6 +112,12 @@
     "package-json": "^8.1.0",
     "typescript": "4.6.4",
     "sharp": "^0.32.6",
-    "ip": "https://registry.npmjs.org/ip-address/-/ip-address-9.0.5.tgz"
+    "ip": "https://registry.npmjs.org/ip-address/-/ip-address-9.0.5.tgz",
+    "**/express/path-to-regexp": "0.1.10",
+    "**/serve/serve-handler/path-to-regexp": "3.3.0",
```

Comment:
Are these docs dependencies? the resolution for docs package should live in its own package.json

Reply:
This PR is a cherry pick from OHIF upstream